crates/matrix-sdk-base/src/store/migration_helpers.rs

```diff
@@ -80,6 +80,8 @@ pub struct RoomInfoV1 {
     #[cfg(feature = "experimental-sliding-sync")]
     latest_event: Option<SyncTimelineEvent>,
     base_info: BaseRoomInfoV1,
+    #[cfg(feature = "experimental-sliding-sync")]
+    recency_timestamp: Option<MilliSecondsSinceUnixEpoch>,
```

Comment:
This is not correct: this struct was used for migrating previous data with a structural change, so instead, you should remove this field here, and fill `recency_timestamp` to `None` in the below constructor of `RoomInfo`.

Reply:
:+1: Done